atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -108,25 +112,26 @@ public void setup() {
         when(runtimeConfig.unresponsiveHostBackoffTimeSeconds()).thenReturn(UNRESPONSIVE_HOST_BACKOFF_SECONDS);
```

Comment:
aside, I'm surprised there are no tests to remove for the token ring consistency validation that we're removing...